x-pack/plugins/security_solution/server/search_strategy/security_solution/factory/risk_score/all/index.ts

```diff
@@ -89,6 +94,7 @@ async function enhanceData(
   return data.map((risk) => ({
     ...risk,
     alertsCount: alertsCountByEntityName[get(nameField, risk)] ?? 0,
+    oldestAlertTimestamp,
```

Comment:
This sets the same `oldestAlertTimestamp` for all the entities in the array, the oldest of all of them. So, for instance, we may have a host with only one alert today, but the `oldestAlertTimestamp` can still be 1 year ago because another host has some alert at that time. Is this it?
If so, would it make sense to put the timestamp in a nested aggregation, so the `oldestAlertTimestamp` in the array will refer to the oldest of that specific entity?
Maybe it does not make sense, just asking.

Reply:
I think that it does make sense. I liked the suggestion, it is a small change, and I just pushed it.